pl_examples/hydra_examples/conf/pl/logger/tensorboard.yaml

```diff
@@ -2,4 +2,4 @@
 cls: pytorch_lightning.loggers.tensorboard.TensorBoardLogger
 params:
   name: ${hydra:job.name}
-  save_dir: /tmp/
+  save_dir: .
```

Comment:
Awesome. 
Document an example of using TB to visualize results from the output directory somewhere.

Reply:
Documented now in readme